### ISSUE TITLE
can't seem to pull docker

### ISSUE THREAD

**Issue by karelbilek** [ryankennedyio/deep-dream-generator#9]
I can't pull the docker. Docker just throws a mysterious error when trying to pull `ipython/ipython`

```
Pulling repository ipython/ipython
4fb8f16a0bfe: Error pulling image (3.x) from ipython/ipython, HTTP code 400 
428b411c28f0: Download complete 
435050075b3f: Download complete 
9fd3c8c9af32: Download complete 
6d4946999d4f: Download complete 
9bd2d4fa7079: Download complete 
d6d97543cd83: Download complete 
7f13ac159046: Download complete 
a15e944e9b19: Download complete 
ee69352e1c03: Error pulling dependent layers
```

Looking at the dockerfile of ipython/ipython, [it's deprecated](https://registry.hub.docker.com/u/ipython/ipython/dockerfile/)

```
# DEPRECATED: You probably want jupyter/notebook
```

I will try to replace ipython/ipython (in your dockerfile) with jupyter/notebook and see what happens. Maybe I'll add a PR, maybe not

**Comment by ryankennedyio** [comment]
@runn1ng Any luck?
I didn't run into this when working on it at all.

**Comment by karelbilek** [comment]
I "fixed" it by installing a new docker version. The one that is included in ubuntu repository - 1.0.1 - just didn't work.
